cli/cmd/lib_cli_config.go

```diff
@@ -149,6 +150,7 @@ func readCLIConfig() (*CLIConfig, []error) {
 func getValidCLIConfig() *CLIConfig {
 	cliConfig, errs := readCLIConfig()
 	if len(errs) > 0 {
+		fmt.Printf("configuring %s environment:\n", flagEnv)
```

Comment:
Sorry for going back and forth, but now I see that `configure()` prints out `"\nEnvironment: " + flagEnv + "\n"`, so I think this can be deleted